packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -921,4 +985,16 @@ private void forceDelete(File file) throws IOException {
       throw new IOException("Unable to delete file: " + file);
     }
   }
+
+  private String importHttpMethod(int method) {
+    if (method == 1) {
+      return "PUT";
+    }
+
+    if (method == 2) {
+      return "PATCH";
+    }
+
+    return "POST";
```

Comment:
I wish we have these magic numbers stored somewhere in global variables or enum 🎉